nixos/tests/ollama.nix

```diff
@@ -32,6 +32,7 @@ in
     };
 
     cuda = { ... }: {
+      nixpkgs.config.allowUnfree = true;
```

Comment:
Maybe we could move https://github.com/NixOS/nixpkgs/blob/0101bf7350903d84558b8059d37c2c1a60482ceb/pkgs/top-level/release-cuda.nix#L16-L28 out into a file to make it more reusable and reduce the need for the blanket-allow

Reply:
I don't really see the benefit, but if you want to push a commit doing that, I'm fine with it.

Reply:
Actually, since this is a pr for ollama targeting stable nixpkgs, it might be better to create a new pr for this and backport it, and then ollama's test can be updated once that's done. But I don't care that much either way, whatever you prefer.

Reply:
Yes, you're right, I'll handle better this in a separate PR, and update the manual at the some time

Reply:
This line might actually be a bad idea altogether according to [this comment](https://github.com/NixOS/nixpkgs/pull/324659#discussion_r1670596585).